lib/env.ts

```diff
@@ -0,0 +1,43 @@
+import { Option, fromNullable } from "fp-ts/lib/Option";
+import * as t from "io-ts";
+
+export const EnvMap = t.exact(
+  t.partial({
+    SPID_SESSION_TOKEN: t.string,
+    IO_BACKEND_HOST: t.string,
+    IO_BACKEND_BASEPATH: t.string
+  })
+);
+export type EnvMap = t.TypeOf<typeof EnvMap>;
+
+const pickEnvKeys = (obj: any) =>
```

Comment:
can we replace the whole methods here with a call to `EnvMap.decode(process.env)` when needed ?